src/webcomponents/variant/interpretation/variant-interpreter-grid.js

```diff
@@ -844,7 +845,12 @@ export default class VariantInterpreterGrid extends LitElement {
                                 // Check if the copy object has an execute function, this prevents two possible scenarios:
                                 // 1. a 'copy' stored in OpenCGA config that has been removed from IVA config
                                 // 2. an incorrect copy configuration
-                                if (copy.execute) {
+
+                                // Ideas:
+                                // CustomActions.check(copy)
+                                // CustomActions.checkVersion(copy)
+                                // CustomActions.get(copy).execute(variant, showConsequenceTypes)
```

Comment:
I think this is not needed there.

Reply:
Nacho´s comments - I would prefer he deletes them if appropriate